packages/@expo/metro-config/src/transform-worker/browserslist.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright © 2025 650 Industries.
+ */
+
+const debug = require('debug')('expo:metro:browserslist') as typeof console.log;
+
+const browserslistCache: Record<string, import('lightningcss').Targets> = {};
+
+export async function getBrowserslistTargets(
+  projectRoot: string
+): Promise<import('lightningcss').Targets> {
+  if (browserslistCache[projectRoot]) {
+    return browserslistCache[projectRoot];
+  }
+  const browserslist = await import('browserslist');
+  const { browserslistToTargets } = require('lightningcss') as typeof import('lightningcss');
+
+  const targets = browserslistToTargets(
+    browserslist.default(undefined, {
```

Comment:
I think this could be a little misleading if we document it since `babel-preset-env` also relies on this. Shouldn't be an issue if people do start relying on it, but just something to be mindful of once or if we add docs for this.

I'm not 100% sure what the default here is. If the default includes IE11, then that could be more than what we're potentially actually supporting?